### PR TITLE
Fix symfony >= 3.4 user deprecated.

### PR DESCRIPTION
**Pull request by instabledesign**
|**Q**|**A**|
| ------------- |:-------------:|
| Branch? | master|
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |

Symfony (v3.4.4)
Fix 3 User Deprecated
https://github.com/symfony/translation/commit/449525b0d30e6f6d32b02c064fdb0e08ae2280c1

image

### CODE REVIEW

**Comment by instabledesign** [comment]
@cedric-g what did you think about it??

File: DependencyInjection/LexikTranslationExtension.php [hunk 1]
```diff
@@ -85,6 +85,14 @@ public function buildTranslatorDefinition(ContainerBuilder $container)
                 new Parameter('lexik_translation.translator.options')
             ];
             $translator->setPublic(true);
+        } elseif (Kernel::VERSION_ID >= 30400) {
+            $arguments = [
```
Comment (eugenekurasov) [inline DependencyInjection/LexikTranslationExtension.php]: Arguments duplicate like for upper condition Kernel::VERSION_ID >= 40000.

Maybe you change 
```
 if (Kernel::VERSION_ID >= 40000) {
```
to 
```
 if (Kernel::VERSION_ID >= 30400) {
```

**Comment by instabledesign** [comment]
rerun tests

**Comment by instabledesign** [comment]
re run tests

**Comment by instabledesign** [comment]
@spike31 or any maintainer

**Comment by instabledesign** [comment]
reping @spike31

**Pull request merged**